src/browser/components/MainPage.jsx

```diff
@@ -369,6 +369,23 @@ export default class MainPage extends React.Component {
     this.handleSelect(key);
   };
 
+  handleInterTeamLink = (linkUrl) => {
+    let selectedTeam;
+    this.props.teams.forEach((team, index) => {
```

Comment:
maybe we should modify https://github.com/mattermost/desktop/blob/master/src/utils/util.js#L85 to return the index. seems an operation that can be done multiple times throughout the code

Reply:
Yeah I think that's worth doing, I'll make the change.